packages/lesswrong/components/sunshineDashboard/SunshineNewUsersList.jsx

```diff
@@ -0,0 +1,31 @@
+import { Components, registerComponent, withList, withCurrentUser } from 'meteor/vulcan:core';
+import React, { Component } from 'react';
+import Users from 'meteor/vulcan:users';
+
+class SunshineNewUsersList extends Component {
+  render () {
+    const results = this.props.results
```

Comment:
Replace with { results } = this.props syntax. Seems good to be consistent about this.